bigtable/tests/system.py

```diff
@@ -272,6 +272,88 @@ def test_delete_column_family(self):
         # Make sure we have successfully deleted it.
         self.assertEqual(temp_table.list_column_families(), {})
 
+    def test_retry(self):
+        import subprocess, os, stat, platform, ssl
+        from google.cloud.bigtable.client import Client
+        from google.cloud.bigtable.instance import Instance
+        from google.cloud.bigtable.table import Table
+
+        # import for urlopen based on version
+        try:
+            # python 3
+            from urllib.request import urlopen
+        except ImportError:
+            # python 2
+            from urllib2 import urlopen
+
+        TEST_SCRIPT = 'tests/retry_test_script.txt'
+        SERVER_NAME = 'retry_server'
+        SERVER_ZIP = SERVER_NAME + ".tar.gz"
+
+        def download_server():
```

Comment:
This method should verify that the downloaded and extracted file doesn't already exist, so it doesn't re-download it every time a user reruns the tests locally.

Right now, it appears that the download will happen unconditionally, every time.

Reply:
Done

Reply:
I'm sorry, I don't see where this function only downloads the first time, and not every time: did you upload your latest changes?